vendor/vendor-ios/src/main/kotlin/com/malinskiy/marathon/ios/xctestrun/Xctestrun.kt

```diff
@@ -24,41 +24,53 @@ class Xctestrun(inputStream: InputStream) {
         ?: throw IllegalArgumentException("could not parse xctestrun")
 
     // testable target properties
-
+/*
     private val target = PropertyListKey.TargetName(
         propertyList.keys.firstOrNull { it != PropertyListKey.__xctestrun_metadata__.toKeyString() }
             ?: throw IllegalArgumentException("xctestrun file does not define any testable targets")
     )
+*/
+
+    private val targetMap: Map<String, *> = (
+        ((((((propertyList.valueForKeypath("TestConfigurations") as? Array<Any>)?.first()) as Map<*, *>)
```

Comment:
first() will throw NullPointerException here when `TestConfigurations` is missing. Reproduced on the old sample